### DISCUSSION TITLE
Number of frames recorded by bonsai does not equal number of TTLs received

### DISCUSSION THREAD

**Discussion by lillysanchez** [bonsai-rx/bonsai#1008] [Q&A]
I am using Bonsai to track the position of an LED. I will be recording the LFP with OpenEphys. My workflow is attached:

![image](https://user-images.githubusercontent.com/101027233/184559746-c0c3ab29-8df3-4f03-9882-96320aedd478.png)

1. I first start OpenEphys
2. I begin capturing with Bonsai, using the PylonCapture node which controls a camera that emits TTLs directly to OpenEphys hardware (not Bonsai).
3. Inspecting the outputs shows that the number of TTLs is not equal to the number of frames captured by Bonsai. This assumes TTL/2 is the number of frames captured (currently 60Hz, so I'd think the computer could keep up with that number of frames)

Why is bonsai skipping frames?

**Comment by chiyu1203** [comment]
Hi I am interested in this question. Have you figured out what happened to the skipping frames? Would you be able to control the camera via Bonsai?! I thought Bonsai just received images via Pylon capture and you can only control the camera with camera sending out TTL pulse. But I dont see such a module in your workflow.
